nbs/directives.ipynb

```diff
@@ -0,0 +1,828 @@
+{
```

Comment:
<p>I would hide the input cell with <code>#| echo: false</code> so that it renders nicely in the docs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>